### ISSUE TITLE
putScript function

### ISSUE THREAD

**Issue by AndrewCherabaev** [elastic/elasticsearch-php#690]
According to [new documentation of templating and sripting](https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking_60_scripting_changes.html#_stored_search_template_apis_removed), could you please edit putScript function or putScript Endpoint to doing next (in Guzzle terms): 

```php
$client = new \GuzzleHttp\Client();
$client->put( "/_scripts/$id", [
    "body" => json_encode([
        "script" => [
            "lang" =>  "$lang",
            "source" => "$body"
        ]
    ])
]);
```

Same problem with `"Invalid index name [_search], must not start with '_', '-', or '+'"` was described in [issue 686](https://github.com/elastic/elasticsearch-php/issues/686)

**Comment by polyfractal** [comment]
Please don't open duplicate issues :)  Makes it hard to track what needs fixing.

This issue was fixed, as described in https://github.com/elastic/elasticsearch-php/issues/686#issuecomment-348212032

**Comment by polyfractal** [comment]
FYI this was just released as part of `v6.0.1` :)
